EXTENDED_VERBS.s

```diff
@@ -76,7 +76,7 @@ LST2FAN		TC	VBZERO		# VB40 ZERO (USED WITH NOUN 20 ONLY)
 		TC	V62		# VB62 SELECT MODE II, ERROR WRT N22
 		TC	V63		# VB63 SELECT MODE III, ERROR WRT N17
 		TC	VB64		# VB64 CALCULATE, DISPLAY S-BAND ANT ANGLES
-		TC	CKOPTVB		# V 65 E OPTICAL VERIFICATION FOR PRELAUNC
+		TC	CKOPTVB		# V 65 E OPTICAL VERIFICATION FOR PRELAUNCH
```

Comment:
`RELAUNC` is what was used in [source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0236.jpg)

Finally clocked on that these missing last character typos are simply because of the character-per-line limit.

![](http://i.imgur.com/HqkzFfG.png)